mw-webapp/src/dataAccessLogic/CurrentProblemDAL.ts

```diff
@@ -24,25 +42,30 @@ export class CurrentProblemDAL {
   /**
    * Create CurrentProblem
    */
-  public static async createCurrentProblem(): Promise<CurrentProblem> {
+  public static async createCurrentProblem(dayReportUuid: string): Promise<CurrentProblem> {
     const currentProblemWithoutUuid: CurrentProblemDTOWithoutUuid = {
-      description: SPACE,
+      description: unicodeSymbols.space,
```

Comment:
enums could be used as types. So for enums use `ALL_CAPITAL_LETTERS` or `UpperCase`

Reply:
I will rewrite but other enums in the project written with lowercase

Reply:
Done